eslint.config.mjs

```diff
@@ -210,12 +210,6 @@ const baseRules = {
 };
 
 const reactRules = {
-  /**
-   * Custom
-   */
-  // highlights literals in JSX components w/o translation tags
-  'getsentry/jsx-needs-il8n': ['off'],
```

Comment:
removed, no need to mention something that's off. And the repo is archived because the rule is buggy